lighter/utils/misc.py

```diff
@@ -6,31 +6,31 @@
 from loguru import logger
 
 
-def ensure_list(x: Any) -> List:
+def ensure_list(vals: Any) -> List:
```

Comment:
Any reason to change it to `vals`? I think `x` looked more readable for quick functions like this

Reply:
Ahh, no worries then, lgtm